<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QDialog" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="leRoot">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>755</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>757</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Root path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConnect">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAddr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPort">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>1234</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDisconnect">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QListView" name="lvSendingFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddSend">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>582</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStartSend">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>780</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Send&amp;&amp;Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClearAll">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QListView" name="lvDeletingFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddDelete">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>613</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClearDelete">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbListFiles">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>750</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>List models</string>
   </property>
  </widget>
  <widget class="QListView" name="lvListFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>584</y>
     <width>421</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>782</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temp path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTemp">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>780</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Передать на сервер</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить с сервера</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>405</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Модели на сервере</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Прочитать с сервера</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>405</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Модели на локальном</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClearReceiveAll">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>60</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClearDeleteLocal">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>260</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddLocalDelete">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>612</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Delete</string>
   </property>
  </widget>
  <widget class="QListView" name="lvLocalDeletingFiles">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="lvLocalListFiles">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>584</y>
     <width>421</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddReceive">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>583</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Receive</string>
   </property>
  </widget>
  <widget class="QListView" name="lvReceiveFiles">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить с локального</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStartReceive">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>780</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Rec&amp;&amp;Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLocalListFiles">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>750</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>List models</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUpdateStruct">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>770</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>770</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QListView" name="lwListModels">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="lwLocalListModels">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>564</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Файлы модели на сервере</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>562</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Файлы модели на клиенте</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Receive Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Model</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbConnect</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDisconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnDisconnecting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddSend</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnAddSend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1041</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbStartSend</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnStartSend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClearAll</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnClearSendAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddDelete</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnAddDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClearDelete</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnClearDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbListFiles</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnListFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddReceive</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnAddReceive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>946</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClearReceiveAll</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnReceiveClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1026</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>1010</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddLocalDelete</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnDeleteLocal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>996</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>982</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClearDeleteLocal</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnClearDeleteLocal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>996</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbStartReceive</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnStartReceive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1016</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>1004</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbLocalListFiles</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnListFilesLocal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>885</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>928</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbUpdateStruct</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnUpdateStruct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>OnClearModels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwLocalListModels</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainForm</receiver>
   <slot>OnLocalModelClick(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>974</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwListModels</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainForm</receiver>
   <slot>OnServerModelClick(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnConnect()</slot>
  <slot>OnProba()</slot>
  <slot>OnDisconnecting()</slot>
  <slot>OnAddSend()</slot>
  <slot>OnStartSend()</slot>
  <slot>OnClearSendAll()</slot>
  <slot>OnAddDelete()</slot>
  <slot>OnStartDelete()</slot>
  <slot>OnClearDelete()</slot>
  <slot>OnListFiles()</slot>
  <slot>OnAddReceive()</slot>
  <slot>OnReceiveClear()</slot>
  <slot>OnDeleteLocal()</slot>
  <slot>OnClearDeleteLocal()</slot>
  <slot>OnStartReceive()</slot>
  <slot>OnListFilesLocal()</slot>
  <slot>OnUpdateStruct()</slot>
  <slot>OnClearModels()</slot>
  <slot>OnLocalModelClick(QModelIndex)</slot>
  <slot>OnServerModelClick(QModelIndex)</slot>
 </slots>
</ui>
